docs/projects/adrv9361z7035/index.rst

```diff
@@ -0,0 +1,365 @@
+.. _adrv9361z7035:
+
+ADRV9361Z7035 HDL Project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`ADRV9361-Z7035` is built on a portfolio of highly integrated
+System-On-Module (SOMs) based on the Xilinx Zynq®-7000 All Programmable (AP)SoC.
```

Comment:
Not entirely sure about the sign, if it will work in the PDF doc., so better to remove it.
```suggestion
System-On-Module (SOMs) based on the Xilinx Zynq-7000 All Programmable (AP) SoC.
```